src/main/java/chessmaster/game/ChessBoard.java

```diff
@@ -153,7 +153,7 @@ public void setPromotionPiece(Coordinate coord, ChessPiece promotedPiece) {
     }
 
     public void setTile(int row, int col, ChessTile tile) {
-        board[row][col] = tile;
+        board[col][row] = tile;
```

Comment:
Looks like this method is not referenced anywhere... help double confirm and delete if so